packages/stacked_services/lib/src/snackbar/snackbar_service.dart

```diff
@@ -68,31 +68,30 @@ class SnackbarService {
     Get.snackbar(
       title,
       message,
-      titleText: _snackbarConfig?.titleColor != null || title.isNotEmpty
+      titleText: title.isNotEmpty
           ? Text(
               title,
               key: Key('snackbar_text_title'),
               style: TextStyle(
-                color: _snackbarConfig?.titleColor,
+                color: _snackbarConfig?.titleColor ?? _snackbarConfig?.textColor ?? Colors.white,
                 fontWeight: FontWeight.w800,
                 fontSize: 16,
               ),
               textAlign: _snackbarConfig?.titleTextAlign ?? TextAlign.left,
             )
           : SizedBox.shrink(),
-      messageText: _snackbarConfig?.messageColor != null || message.isNotEmpty
+      messageText: message.isNotEmpty
           ? Text(
               message,
               key: Key('snackbar_text_message'),
               style: TextStyle(
-                color: _snackbarConfig?.messageColor,
+                color: _snackbarConfig?.messageColor ?? _snackbarConfig?.textColor ?? Colors.white,
                 fontWeight: FontWeight.w300,
                 fontSize: 14,
               ),
               textAlign: _snackbarConfig?.messageTextAlign ?? TextAlign.left,
             )
           : SizedBox.shrink(),
-      colorText: _snackbarConfig?.textColor ?? Colors.white,
```

Comment:
This field is not used because we are always setting the title and message style widget explicitly